src/order/order_hom.lean

```diff
@@ -10,69 +10,69 @@ import order.bounded_order
-* `preorder_hom.fst`: projection `prod.fst : α × β → α` as a bundled monotone map;
-* `preorder_hom.snd`: projection `prod.snd : α × β → β` as a bundled monotone map;
-* `preorder_hom.prod_map`: `prod.map f g` as a bundled monotone map;
-* `pi.eval_preorder_hom`: evaluation of a function at a point `function.eval i` as a bundled
+* `order_hom.id`: identity map as `α →ₘ α`;
+* `order_hom.curry`: an order isomorphism between `α × β →ₘ γ` and `α →ₘ β →ₘ γ`;
+* `order_hom.comp`: composition of two bundled monotone maps;
+* `order_hom.compₘ`: composition of bundled monotone maps as a bundled monotone map;
+* `order_hom.const`: constant function as a bundled monotone map;
+* `order_hom.prod`: combine `α →ₘ β` and `α →ₘ γ` into `α →ₘ β × γ`;
+* `order_hom.prodₘ`: a more bundled version of `order_hom.prod`;
+* `order_hom.prod_iso`: order isomorphism between `α →ₘ β × γ` and `(α →ₘ β) × (α →ₘ γ)`;
+* `order_hom.diag`: diagonal embedding of `α` into `α × α` as a bundled monotone map;
+* `order_hom.on_diag`: restrict a monotone map `α →ₘ α →ₘ β` to the diagonal;
+* `order_hom.fst`: projection `prod.fst : α × β → α` as a bundled monotone map;
+* `order_hom.snd`: projection `prod.snd : α × β → β` as a bundled monotone map;
+* `order_hom.prod_map`: `prod.map f g` as a bundled monotone map;
+* `pi.eval_order_hom`: evaluation of a function at a point `function.eval i` as a bundled
   monotone map;
-* `preorder_hom.coe_fn_hom`: coercion to function as a bundled monotone map;
-* `preorder_hom.apply`: application of a `preorder_hom` at a point as a bundled monotone map;
-* `preorder_hom.pi`: combine a family of monotone maps `f i : α →ₘ π i` into a monotone map
+* `order_hom.coe_fn_hom`: coercion to function as a bundled monotone map;
+* `order_hom.apply`: application of a `order_hom` at a point as a bundled monotone map;
+* `order_hom.pi`: combine a family of monotone maps `f i : α →ₘ π i` into a monotone map
   `α →ₘ Π i, π i`;
-* `preorder_hom.pi_iso`: order isomorphism between `α →ₘ Π i, π i` and `Π i, α →ₘ π i`;
-* `preorder_hom.subtyle.val`: embedding `subtype.val : subtype p → α` as a bundled monotone map;
-* `preorder_hom.dual`: reinterpret a monotone map `α →ₘ β` as a monotone map
+* `order_hom.pi_iso`: order isomorphism between `α →ₘ Π i, π i` and `Π i, α →ₘ π i`;
+* `order_hom.subtyle.val`: embedding `subtype.val : subtype p → α` as a bundled monotone map;
+* `order_hom.dual`: reinterpret a monotone map `α →ₘ β` as a monotone map
   `order_dual α →ₘ order_dual β`;
-* `preorder_hom.dual_iso`: order isomorphism between `α →ₘ β` and
+* `order_hom.dual_iso`: order isomorphism between `α →ₘ β` and
   `order_dual (order_dual α →ₘ order_dual β)`;
 
 We also define two functions to convert other bundled maps to `α →ₘ β`:
 
-* `order_embedding.to_preorder_hom`: convert `α ↪o β` to `α →ₘ β`;
-* `rel_hom.to_preorder_hom`: conver a `rel_hom` between strict orders to a `preorder_hom`.
+* `order_embedding.to_order_hom`: convert `α ↪o β` to `α →ₘ β`;
```

Comment:
Just pointing out that notation is inconsistent. To homogeneize, I'd vote for replacing `α →ₘ β` by `α →o β`.
Of course, no need for that to be in this PR.

Reply:
I vote to do that in a later PR, I'm currently moving around a bunch of files (as you know :D)